server/sonar-db-dao/src/test/java/org/sonar/db/component/ComponentDaoTest.java

```diff
@@ -1281,18 +1282,18 @@ public void selectAncestors() {
   }
 
   @Test
-  public void select_descendants_with_children_stragegy() {
+  public void select_descendants_with_children_strategy() {
     // project has 2 children: module and file 1. Other files are part of module.
     ComponentDto project = newPrivateProjectDto(db.organizations().insert(), PROJECT_UUID);
     db.components().insertProjectAndSnapshot(project);
     ComponentDto module = newModuleDto(MODULE_UUID, project);
     db.components().insertComponent(module);
-    ComponentDto file1 = newFileDto(project, null, FILE_1_UUID).setDbKey("file-key-1").setName("File One");
-    db.components().insertComponent(file1);
-    ComponentDto file2 = newFileDto(module, null, FILE_2_UUID).setDbKey("file-key-2").setName("File Two");
-    db.components().insertComponent(file2);
-    ComponentDto file3 = newFileDto(module, null, FILE_3_UUID).setDbKey("file-key-3").setName("File Three");
-    db.components().insertComponent(file3);
+    ComponentDto fileInProject = newFileDto(project, null, FILE_1_UUID).setDbKey("file-key-1").setName("File One");
+    db.components().insertComponent(fileInProject);
```

Comment:
This 2 lines could have been inlined : 
`ComponentDto fileInProject = db.components().insertComponent(newFileDto(project, null, FILE_1_UUID).setDbKey("file-key-1").setName("File One"));`
Same for 2 following variables